ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Wizard/ProfileSelection.tsx

```diff
@@ -49,61 +69,89 @@ function ProfileSelection({ profiles, isFetchingProfiles }: ProfileSelectionProp
-                <Td>{description}</Td>
-            </Tr>
+        return profiles?.map(({ description, name, productType, ruleCount, title }, rowIndex) => (
+            <Tbody>
+                <Tr key={name}>
+                    <Td
+                        key={name}
+                        select={{
+                            rowIndex,
+                            onSelect: (event, isSelected) =>
+                                handleSelect(event, isSelected, rowIndex),
+                            isSelected: selected[rowIndex],
+                        }}
+                    />
+                    <Td
+                        expand={{
+                            rowIndex,
+                            isExpanded: isProfileExpanded(name),
+                            onToggle: () => setProfileExpanded(name, !isProfileExpanded(name)),
+                        }}
+                    />
+                    <Td>{name}</Td>
+                    <Td>{ruleCount}</Td>
+                    <Td>{productType}</Td>
+                </Tr>
+                <Tr isExpanded={isProfileExpanded(name)}>
+                    <Td colSpan={1}></Td>
+                    <Td colSpan={1}></Td>
+                    <Td dataLabel="" colSpan={3}>
+                        <ExpandableRowContent>
+                            <Text className="pf-u-font-weight-bold">{title}</Text>
+                            <Divider component="div" className="pf-u-my-md" />
+                            <Text>{description}</Text>
+                        </ExpandableRowContent>
+                    </Td>
+                </Tr>
+            </Tbody>
         ));
     }
 
     function renderLoadingContent() {
         return (
-            <Tr>
-                <Td colSpan={3}>
-                    <Bullseye>
-                        <Spinner isSVG />
-                    </Bullseye>
-                </Td>
-            </Tr>
+            <Tbody>
+                <Tr>
+                    <Td colSpan={5}>
+                        <Bullseye>
+                            <Spinner isSVG />
+                        </Bullseye>
+                    </Td>
+                </Tr>
+            </Tbody>
         );
     }
 
     function renderEmptyContent() {
         return (
-            <Tr>
-                <Td colSpan={3}>
-                    <Bullseye>
-                        <EmptyStateTemplate
-                            title="No profiles"
-                            headingLevel="h3"
-                            icon={SearchIcon}
-                        />
-                    </Bullseye>
-                </Td>
-            </Tr>
+            <Tbody>
+                <Tr>
+                    <Td colSpan={5}>
+                        <Bullseye>
+                            <EmptyStateTemplate
+                                title="No profiles"
+                                headingLevel="h3"
+                                icon={SearchIcon}
```

Comment:
Something to consider, but not a blocker to merge: `SearchIcon` suggests no profiles **found** that match a filter. Is there a realistic scenario in which there might be no profiles, period. For example, it is plausible for administration events or discovered clusters, so they distinguish. An rough edge is lack of general purpose icon for the none exist versus none found.